Frends.GoogleBigQuery.ListResource/Frends.GoogleBigQuery.ListResource/Definitions/Result.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace Frends.GoogleBigQuery.ListResource.Definitions;
+
+/// <summary>
+/// Task's result.
+/// </summary>
+public class Result
+{
+    /// <summary>
+    /// Operation complete without errors.
+    /// </summary>
+    /// <example>true</example>
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// Data.
+    /// </summary>
+    /// <example>{{  "datasets": [ { "datasetReference": { "datasetId": "tasktest", "projectId": "instant-stone-387712",... }}</example>
+    public JToken Data { get; private set; }
```

Comment:
Change the type to dynamic. If this is JToken dot-notation will not work in Frends.